frontend/src/entry/model/Files.tsx

```diff
@@ -14,13 +16,58 @@ type FilesProps = {
 
 export default function Files({ model }: FilesProps) {
   const { entryFiles, isEntryFilesLoading, isEntryFilesError } = useGetModelFiles(model.id)
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
+  const menuOpen = Boolean(anchorEl)
+  const [orderByValue, setOrderByValue] = useState('createdAt')
+  const [orderByButtonTitle, setOrderByButtonTitle] = useState('Order by')
+  const [ASCorDESC, setASCorDESC] = useState('DESC')
```

Comment:
I think we should type this:

```
export const SortingDirection= {
  ASC: 'ascending',
  DESC: 'descending',
} as const
export type SortingDirectionKeys = (typeof SortingDirection)[keyof typeof SortingDirection]
```

That way, we can use it like so:

 `const [ascOrDesc, setAscOrDesc] = useState<SortingDirectionKeys>(SortingDirection.DESC)`

In the above I also changed `ASCorDESC` to `ascOrDesc` as the casing is more consistent